src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Color {
+    RED,
+    GREEN,
+    BLUE,
+    BLACK,
+    WHITE,
+    PURPLE
+}
+
```

Comment:
```suggestion
```
let's make sure there's exactly one empty line at the end of each file, you have two here

Reply:
Fixed